web/src/lib/domain-logic/starterKits.test.ts

```diff
@@ -0,0 +1,64 @@
+import { EssentialQuestions } from "@/lib/domain-logic/essentialQuestions";
+import { createStarterKitProfileData } from "@/lib/domain-logic/starterKits";
+import { emptyProfileData, Industries, ProfileData } from "@businessnjgovnavigator/shared";
+
+describe("starter Kits", () => {
+  const combineObjectsKeepingTruthyValues = (obj1: ProfileData, obj2: ProfileData): ProfileData => {
+    let result: ProfileData = { ...obj1 };
+    for (const key in obj2) {
+      const typedKey = key as keyof ProfileData;
+      if (!result[typedKey]) {
+        result = {
```

Comment:
dust - you could update logic to a single line
`result[typedKey] = obj2[typedKey]`

Reply:
I'm pretty sure I can't because the field is read only, I had it initially this way but I was getting errors